vms/volunteer/migrations/verified_user_migration.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
No, pls remove this. We don't push migrations since it's still under development and that too by a lot of contributors